dq-game/src/client/client_controller.py

```diff
@@ -17,7 +18,32 @@ def __init__(self, socket, renderer, screen_handler):
     def start_game(self):
         self.trigger("controller_start_game")
 
+    def get_theme_list(self):
+        """
+        Gets the theme list from the inbuffer and returns False if not found
+        """
+        for msg in self.socket.inbuffer:
+            if msg.content_type == "data-theme-list":
+                return msg.data
+        return False
+
     def get_client_theme_choices(self):
         # Send instruction to choose themes to clients
-        print('client controller get client theme choices')
-        self.renderer.select_themes(mock_themes)
+        print("client controller get client theme choices")
+        theme_list = self.get_theme_list()
+
```

Comment:
This piece of code chooses 3 random themes from the list in order to test functionality, this is the one you need to replace with View IO code.